revolv/payments/tests/models/test_payment.py

```diff
@@ -56,46 +70,169 @@ def test_donations(self):
         project2 = Project.factories.base.create()
 
         self._create_payment(user1).save()
-        self._create_payment(user1, instrument_type=self.reinvestment).save()
-        self._create_payment(user1, instrument_type=self.repayment).save()
-
-        self._create_payment(user2, instrument_type=self.repayment).save()
+        self._create_payment(user1, payment_type=self.reinvestment).save()
 
         self.assertEquals(Payment.objects.donations(user1).count(), 1)
         self.assertEquals(Payment.objects.donations(user1, project2).count(), 0)
 
-        self._create_payment(user1, project2).save()
+        self._create_payment(user1, project=project2).save()
         self.assertEquals(Payment.objects.donations(user1, project2).count(), 1)
 
         self.assertEquals(Payment.objects.donations(user2).count(), 0)
 
-    def test_reinvestments(self):
-        """Test that we can bookkeep reinvestment information."""
+    def test_proportion_donated(self):
+        """Verify that repayment proportions are correct."""
+        user1, user2, admin1 = RevolvUserProfile.factories.base.create_batch(3)
```

Comment:
If this needs to be an admin, then you should at least do `RevolvUserProfile.factories.admin.create()` instead of `base`